src/engine/strat_engine/blockdev.rs

```diff
@@ -231,7 +248,7 @@ impl BlockDev {
         }
     }
 
-    pub fn wipe_metadata(&mut self) -> EngineResult<()> {
+    pub fn wipe_metadata(&self) -> EngineResult<()> {
```

Comment:
I'm wondering if we actually want this to take `self`, i.e. consume the blockdev. Would this work?